edx_proctoring/backends/__init__.py

```diff
@@ -16,6 +16,6 @@ def get_backend_provider(exam=None, name=None):
         if 'is_proctored' in exam and not exam['is_proctored']:
             # timed exams don't have a backend
             return None
-        elif exam['backend']:
```

Comment:
whats the reason for removing elif ?

Reply:
According to pylint if you shouldn't use else after a return in if block.

https://pheanex.github.io/pylint/